test/intl402/Temporal/PlainDate/prototype/since/wrapping-at-end-of-month-indian.js

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.prototype.since
+description: Tests balancing of days to months at end of month (indian calendar)
+includes: [temporalHelpers.js]
+features: [Temporal, Intl.Era-monthcode]
+---*/
+
+const calendar = "indian";
+
+// Difference between end of longer month to end of following shorter month
+{
+  const end = Temporal.PlainDate.from({ year: 1970, monthCode: "M07", day: 30, calendar });
```

Comment:
Let's use years in the recent past for the calendar, like 1945, and 1942 for the leap year.